### PR TITLE
Created flash_finder command 

### PR DESCRIPTION
**Pull request by anushka255**
- Fixed the two bugs in `flash_finder`
- Converted `flash_finder` into a wbi command 
- Removed two failing test cases for the bugs that are now fixed

### CODE REVIEW

File: tests/test_legacy_flash_finder.py [hunk 1]
```diff
@@ -20,21 +19,3 @@ def test_flash_finder():
         assert os.path.exists(os.path.join(temp_dir, "hiResData.mat"))
         assert os.path.exists(os.path.join(temp_dir, "CameraFrameData.txt"))
         assert os.path.exists(os.path.join(temp_dir, "submissionParameters.txt"))
-
-
-def test_flash_bug1():
```
Comment (vineetbansal) [inline tests/test_legacy_flash_finder.py:25]: When fixing a bug, the test that revealed the bug should remain in place, but the only line that should get removed is the `with pytest.raises(..):`, and maybe add a comment on the test that this is fixed. That way we can prove to ourselves that the bug was fixed. (The redundant test case can be removed in a later PR).

File: src/wbi/commands/flash_finder.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        "--output_folder",
+        type=str,
+        required=True,
+        help="Path to the folder to save output files",
+    )
+    parser.add_argument(
+        "--chunksize", type=int, required=False, help="Approximate number of frames"
+    )
+    parser.add_argument(
+        "-v", "--verbose", action="store_true", help="Increase verbosity"
```
Comment (vineetbansal) [inline src/wbi/commands/flash_finder.py:25]: This is not used anywhere. Just set the logging level appropriately here based on this flag, just like the current hello/goodbye commands, before those 2 sample commands are gone.

File: src/wbi/legacy/flash_finder.py [hunk 3]
```diff
@@ -40,11 +40,19 @@ def flash_finder(input_folder, output_folder, chunksize=4000):
...
+
+    # In cases where the number of frames in .dat file aren't exact multiple of 6 * chunk_size
+    if remainingframes > 0:
+        # Last partial chunk
+        chunk = np.fromfile(f, dtype=np.uint16, count=remainingframes * 1024 * 512)
+        frames = chunk.reshape((remainingframes, 1024 * 512))
+        try:
+            brightness[-remainingframes - 1:-1] = np.average(frames, axis=1)
+            stdev[-remainingframes - 1:-1] = np.std(frames.astype(np.float64), axis=1)
+        except ValueError:
```
Comment (vineetbansal) [inline src/wbi/legacy/flash_finder.py]: where in lines 50/51 do we expect a `ValueError` to be raised?

Comment (anushka255) [reply src/wbi/legacy/flash_finder.py]: Both of those lines will cause `ValueError` in cases where `chunksize // 6 ` is greater than number of frames in .dat file (in this case is 90). In both of these lines the shape of the slice from the either `brightness` or `stdev` array does not match the shape of the array resulting from `np.average(frames, axis=1)` or `np.std(frames, axis=1)`.

File: tests/test_legacy_flash_finder.py (hunk 1 above)
Comment (anushka255) [reply tests/test_legacy_flash_finder.py:25]: Reverted back to the original test bases for both bugs with the only change being the removal of `with pytest.raises(..):`

File: src/wbi/commands/flash_finder.py (hunk 2 above)
Comment (anushka255) [reply src/wbi/commands/flash_finder.py:25]: Added a very basic logging level information.

**Pull request merged**